History.md

```diff
@@ -119,107 +121,107 @@ Each patchlevel release contains a separate security fix. We recommend simply up
-  * Add extconf tests for DTLS_method & TLS_server_method, use in minissl.rb. Should fix "undefined symbol: DTLS_method" when compiling against old OpenSSL versions. (#1832)
+  * Fix socket removed after reload - should fix problems with systemd socket activation. ([#1829])
+  * Add extconf tests for DTLS_method & TLS_server_method, use in minissl.rb. Should fix "undefined symbol: DTLS_method" when compiling against old OpenSSL versions. ([#1832])
 * 1 other
-  * Removed unnecessary RUBY_VERSION checks. (#1827)
+  * Removed unnecessary RUBY_VERSION checks. ([#1827])
 
 ## 4.0.0 / 2019-06-25
 
 * 9 features
-  * Add support for disabling TLSv1.0 (#1562)
-  * Request body read time metric (#1569)
-  * Add out_of_band hook (#1648)
-  * Re-implement (native) IOBuffer for JRuby (#1691)
-  * Min worker timeout (#1716)
-  * Add option to suppress SignalException on SIGTERM (#1690)
-  * Allow mutual TLS CA to be set using `ssl_bind` DSL (#1689)
-  * Reactor now uses nio4r instead of `select` (#1728)
-  * Add status to pumactl with pidfile (#1824)
+  * Add support for disabling TLSv1.0 ([#1562])
+  * Request body read time metric ([#1569])
+  * Add out_of_band hook ([#1648])
+  * Re-implement (native) IOBuffer for JRuby ([#1691])
+  * Min worker timeout ([#1716])
+  * Add option to suppress SignalException on SIGTERM ([#1690])
+  * Allow mutual TLS CA to be set using `ssl_bind` DSL ([#1689])
+  * Reactor now uses nio4r instead of `select` ([#1728])
+  * Add status to pumactl with pidfile ([#1824])
 
 * 10 bugfixes
-  * Do not accept new requests on shutdown (#1685, #1808)
-  * Fix 3 corner cases when request body is chunked (#1508)
-  * Change pid existence check's condition branches (#1650)
-  * Don't call .stop on a server that doesn't exist (#1655)
-  * Implemented NID_X9_62_prime256v1 (P-256) curve over P-521 (#1671)
-  * Fix @notify.close can't modify frozen IOError (RuntimeError) (#1583)
-  * Fix Java 8 support (#1773)
-  * Fix error `uninitialized constant Puma::Cluster` (#1731)
-  * Fix `not_token` being able to be set to true (#1803)
-  * Fix "Hang on SIGTERM with ruby 2.6 in clustered mode" ([PR #1741], [#1674], [#1720], [#1730], [#1755])
-
-[PR #1741]: https://github.com/puma/puma/pull/1741
+  * Do not accept new requests on shutdown ([#1685], [#1808])
+  * Fix 3 corner cases when request body is chunked ([#1508])
+  * Change pid existence check's condition branches ([#1650])
+  * Don't call .stop on a server that doesn't exist ([#1655])
+  * Implemented NID_X9_62_prime256v1 (P-256) curve over P-521 ([#1671])
+  * Fix @notify.close can't modify frozen IOError (RuntimeError) ([#1583])
+  * Fix Java 8 support ([#1773])
+  * Fix error `uninitialized constant Puma::Cluster` ([#1731])
+  * Fix `not_token` being able to be set to true ([#1803])
+  * Fix "Hang on SIGTERM with ruby 2.6 in clustered mode" ([PR [#1741]], [#1674], [#1720], [#1730], [#1755])
+
+[PR [#1741]]: https://github.com/puma/puma/pull/1741
```

Comment:
Not sure what's wrong, but the rendered file doesn't look good here:

<img width="876" alt="Screenshot 2020-09-25 at 07 52 37" src="https://user-images.githubusercontent.com/42626/94231426-4c1ad380-ff04-11ea-8607-261af5cbab09.png">